packages/components/stories/ListComposition/ListComposition.js

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import talendIcons from '@talend/icons/dist/react';
+
+import { simpleCollection } from './collection';
+import { IconsProvider } from '../../src/index';
+import List from '../../src/List/ListComposition';
+import CellTitle from '../../src/VirtualizedList/CellTitle';
+import CellBadge from '../../src/VirtualizedList/CellBadge';
+
+const icons = {
```

Comment:
Would be great to not have this icons to be defined for this story ( but i don't care if it's not modified for now )

Reply:
I removed it